server/api-doc.md

```diff
@@ -0,0 +1,485 @@
+# Fancy Todo
+
+## 1. User
+
+### Register
+
+- **URL** : `/register`
+- **Method** : `POST`
+- **Data Params** :
+    ```
+    email=[string]
+    password=[string]
+    ```
```

Comment:
klo kyk gini akan dikira array of string, jadi mending dibuat `string` gitu aja tanpa `[]`